### PR TITLE
Adding a page on asking for help.

### PR DESCRIPTION
**Pull request by LynHUK**
A simple guide to creating a request for help, including a reprex and editing data to remove anything sensitive or confidential.

### CODE REVIEW

File: asking_for_help.qmd
Comment (Lextuga007) [inline asking_for_help.qmd:1]: Duplicate 🙂 as this is also in the blog folder.

**Review by Lextuga007 — CHANGES_REQUESTED** [review]
Can you delete the duplicate blog? Thanks 🙂

**Comment by LynHUK** [comment]
I was trying to move it from the main folder to the blog folder, but I thought it had just not let me do anything. [🤷‍♀️]



Lyn Howard| AdvFEDIP<https://www.fedip.org/advanced-practitioner-fedip>
Clinical Network Manager
NWC Maternity Clinical Network
Click to call me on Microsoft ***@***.***>

M: 07900 715283
E: ***@***.******@***.***> | W: www.england.nhs.uk<http://www.england.nhs.uk/>

This email is intended only for use by the named addressee. It may contain confidential and/or privileged information. If you are not the intended recipient, you should contact us immediately and should not disclose, use or rely on this email. We do not accept any liability arising from a third party taking action, or refraining from taking action, on the basis of information contained in this email. Thank you.

**Comment by Lextuga007** [comment]
I’ve just discovered you can do a file deletion through GitHub! Go to this page Adding a page on asking for help. by LynHUK · Pull Request #199 · nhs-r-community/nhs-r-community (github.com)<https://github.com/nhs-r-community/nhs-r-community/pull/199/files> and in the top right there are three dots. If you select delete file it will take you to a page to “commit the change” directly through GitHub. That way seems really straightforward and quite cool!

Z


Zoë Turner
Senior Data Scientist
The Strategy Unit
E: ***@***.******@***.***>
[The Strategy Unit logo which is a yellow speech like bubble with the words, in black saying The Strategy Unit with a white full stop.] [The NHS-R Community logo which is NHS-R in blue capitals with Community underneath in light grey capitals. The R is like the logo for the program R with a grey oval behind the letter R.]
Leading research, analysis and change from within the NHS.

E: ***@***.******@***.***>
W: nhsrcommunity.com<https://nhsrcommunity.com/>

[The NHS-R Community logo which is NHS-R in blue capitals with Community underneath in light grey capitals. The R is like the logo for the program R with a grey oval behind the letter R.]
NHS-R/NHS.pycom Open Source Conference (RPySOC 2024)
We are excited to announce we are now calling for abstracts.
For full information: https://nhsrway.nhsrcommunity.com/conference.html#rpysoc-24-conference---call-for-abstracts

Tickets still are available! https://nhsrcommunity.com/events/#event_type-conferences

**Review by Lextuga007 — APPROVED** [review]

**Comment by Lextuga007** [comment]
I removed your signature information from the email that appears in the comments - just so you know 🙂

**Comment by Lextuga007** [comment]
@all-contributors
Please add @LynHUK for blog

**Pull request merged**